lms/templates/courseware/courses.html

```diff
@@ -7,28 +7,25 @@
 
 <%namespace name='static' file='../static_content.html'/>
 
+% if settings.FEATURES.get('ENABLE_COURSE_DISCOVERY'):
 <%block name="header_extras">
-  % if settings.FEATURES.get('ENABLE_COURSE_DISCOVERY'):
   % for template_name in ["result_item", "filter_bar", "filter", "search_facets_list", "search_facets_section", "search_facet", "more_less_links"]:
   <script type="text/template" id="${template_name}-tpl">
       <%static:include path="discovery/${template_name}.underscore" />
   </script>
   % endfor
-  <script type="text/javascript">;(function (define) {{
-      define('course_discovery_meanings', function() {{
-          'use strict';
+  <script type="text/javascript">(function (define) {
+      'use strict';
+      define('course_discovery_meanings', function() {
```

Comment:
I know this isn't introduced by this PR, but the way we've handled this in our other views is to pass the ``json.dumps`` result to the factory directly, rather than defining a faux RequireJS dependency. I think it would be simpler to read and easier to understand. It would also make it easier to test, as you can just pass a set of meanings to the factory, rather than having to define a dependency. 

Here's an example from the teams tab:

https://github.com/edx/edx-platform/blob/master/lms/djangoapps/teams/templates/teams/teams.html#L27

Note that this code also escapes the ``json.dumps`` call to avoid XSS attacks.

I'll add a description of our best practice to the wiki page that I'm writing. If it is simple I'd argue for making the same change here, but if it will complicate things too much then it can wait.